src/python/pants/backend/visibility/rule_types_test.py

```diff
@@ -774,3 +833,120 @@ def test_file_specific_rules(rule_runner: RuleRunner) -> None:
         ("src/lib/pub/exception.txt:../lib", DependencyRuleAction.DENY),
         ("src/lib/priv/secret.txt:../lib", DependencyRuleAction.WARN),
     )
+
+
+@pytest.fixture
+def setup_testbed(rule_runner: RuleRunner) -> bool:
```

Comment:
thought I'd have a ton more tests using this, and maybe I will, but think I should probably get rid of this testbed for now, and just setup the case I'm currently testing (below)